R/fit.R

```diff
@@ -252,11 +252,10 @@ mmrm_control <- function(n_cores = 1L,
 #'   the fitting process. Should be `NULL` or a numeric vector.
 #' @param reml (`flag`)\cr whether restricted maximum likelihood (REML)
 #'   estimation is used, otherwise maximum likelihood (ML) is used.
-#' @param covariance (`cov_struct`)\cr
-#'   A covariance structure type definition as produced with [cov_struct()],
-#'   or value that can be coerced to a covariance structure using
-#'   [as.cov_struct()]. If no value is provided, alstructure is derived from
-#'   the provided formula.
+#' @param covariance (`cov_struct`)\cr A covariance structure type definition
```

Comment:
```suggestion
#' @param covariance (`cov_struct`)\cr a covariance structure type definition
```
after the `@param` etc we start with lower case, end with full stop.

but thanks for getting closer already! almost there

Reply:
`%s/\(@param.*\\cr \)\(.\)/\1\L\2/g`'d all my files.